src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -80,12 +89,12 @@ class CreateSavedSearchButton extends React.Component {
     }
   };
 
-  handleChangeName = val => {
-    this.setState({name: val});
+  handleChangeName = (val: string | number | boolean) => {
+    this.setState({name: val as string});
```

Comment:
maybe you could cast the value where you use this function or maybe you could also do as above?